lldb/source/Plugins/Process/Utility/NativeRegisterContextDBReg_arm64.cpp

```diff
@@ -274,7 +161,7 @@ uint32_t NativeRegisterContextDBReg_arm64::SetHardwareWatchpoint(
   }
 
   // Setup control value
-  control_value = g_enable_bit | g_pac_bits | GetSizeBits(size);
+  control_value = m_hw_dbg_enable_bit | g_pac_bits | GetSizeBits(size);
```

Comment:
Perhaps you can also move this sort of thing into a method to be overridden. Then this code can also move to the base class.

Reply:
I have made updates based on [this](https://github.com/llvm/llvm-project/pull/118770#discussion_r1871170826) feedback.